build/docker.mk

```diff
@@ -57,6 +58,7 @@ endef
 define generateDockerMultiArches
 .PHONY: docker-multi-arch-build-$(1)
 docker-multi-arch-build-$(1): build-$(1)-linux-arm64 build-$(1)-linux-amd64 build-$(1)-linux-arm
+	@echo "docker-multi-arch-build-$(1): build-$(1)-linux-arm64 build-$(1)-linux-amd64 build-$(1)-linux-arm"
```

Comment:
Do we need to print this? how does it help users? it's just a list of targets.

Reply:
No, this was for debugging purposes and I have forgotten this here. I removed it now.